src/colors/cyan.js

```diff
@@ -1,6 +1,6 @@
 const cyan = {
     500: '#00aeef',
-    600: '#0093c6',
+    600: '#1c93c4',
```

Comment:
This is a sweeping change, right?  Just confirming it is what is intended?

Reply:
Yes, this was intended to match what design has in the docs.

Reply:
Plus, not much of a difference

![cyan_600_switcheroo](https://user-images.githubusercontent.com/4349658/106526824-1cc03880-649b-11eb-9875-b3a906caa687.png)